Sources/FuzzilliCli/Profiles/JSCProfile.swift

```diff
@@ -49,8 +49,12 @@ let jscProfile = Profile(
             "--thresholdForFTLOptimizeAfterWarmUp=1000",
             "--thresholdForFTLOptimizeSoon=1000",
             // Enable bounds check elimination validation
-            "--validateBCE=true",
+            //"--validateBCE=true",
```

Comment:
What's the reason for this change? Is that feature only available on linux? I'm not sure how important it even is, maybe we can drop it entirely

Reply:
I guess this just do not support for some old version, so just take care.
Here with the comment of `// Enable bounds check elimination validation`, I think there should be some function for that? So I donot handle with that.
Or just check the macOS sdk version to validate if this can be used for the build version of jsc? (For purpose of the old version? maybe this is not necessary)